pages/FAQ/_index.md

```diff
@@ -62,6 +62,9 @@ Use a keybind (or execute) `grim -g "$(slurp)"`, select a region. A screenshot
 will pop into your `~/Pictures/` (You can configure grim and slurp, see their
 GitHub pages).
 
+If you're wanting those screenshots to go directly to your clipboard, consider using `wl-copy`, from [`wl-clipboard`](https://github.com/bugaevc/wl-clipboard) or even `xclip`, an example binding may look something like this:
+`bind = SUPER_CTRL, S, exec, grim -g "$(slurp -d)" - | wl-copy`
+
```

Comment:
```suggestion
If you want those screenshots to go directly to your clipboard, consider using `wl-copy`, from [`wl-clipboard`](https://github.com/bugaevc/wl-clipboard). Here's an example binding:
`bind = , Print, exec, grim -g "$(slurp -d)" - | wl-copy`
```

`xclip` is an X11 utility, not sure why you'd recommend it.

Reply:
fair